farmdata2_modules/fd2_tabs/fd2_barn_kit/seedingReport.spec.js

```diff
@@ -93,6 +92,112 @@ describe('Testing for the seeding report page', () => {
         })
     })
 
+        context('can see No Logs message at appropriate times', () => {
+        before(() => {
+            //cy.login('manager1', 'farmdata2')
+            //cy.visit('/farm/fd2-barn-kit/seedingReport')
```

Comment:
I think you want the visit here but not the login.  Also should these be a beforeEach or a before?